spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/BindingProperties.java

```diff
@@ -243,6 +245,14 @@ public void setPartitioned(Boolean partitioned) {
 		this.partitioned = partitioned;
 	}
 
+	public Boolean getDurableSubscription() {
```

Comment:
I'll leave it as is for now, since we do `get` with other booleans in the same class. Maybe some cleanup after.